docs/source/basic_tutorials/launcher.md

```diff
@@ -135,57 +189,87 @@ Options:
           [env: MAX_WAITING_TOKENS=]
           [default: 20]
 
+```
+#HOSTNAME
+```shell
       --hostname <HOSTNAME>
           The IP address to listen on
           
           [env: HOSTNAME=]
           [default: 0.0.0.0]
 
+```
+#PORT
+```shell
   -p, --port <PORT>
           The port to listen on
           
           [env: PORT=]
           [default: 3000]
 
+```
+#SHARD_UDS_PATH
+```shell
       --shard-uds-path <SHARD_UDS_PATH>
           The name of the socket for gRPC communication between the webserver and the shards
           
           [env: SHARD_UDS_PATH=]
           [default: /tmp/text-generation-server]
 
+```
+#MASTER_ADDR
+```shell
       --master-addr <MASTER_ADDR>
           The address the master shard will listen on. (setting used by torch distributed)
           
           [env: MASTER_ADDR=]
           [default: localhost]
 
+```
+#MASTER_PORT
+```shell
       --master-port <MASTER_PORT>
           The address the master port will listen on. (setting used by torch distributed)
           
           [env: MASTER_PORT=]
           [default: 29500]
 
+```
+#HUGGINGFACE_HUB_CACHE
+```shell
       --huggingface-hub-cache <HUGGINGFACE_HUB_CACHE>
           The location of the huggingface hub cache. Used to override the location if you want to provide a mounted disk for instance
           
           [env: HUGGINGFACE_HUB_CACHE=]
 
+```
+#WEIGHTS_CACHE_OVERRIDE
+```shell
       --weights-cache-override <WEIGHTS_CACHE_OVERRIDE>
           The location of the huggingface hub cache. Used to override the location if you want to provide a mounted disk for instance
           
           [env: WEIGHTS_CACHE_OVERRIDE=]
 
+```
+#disable-custom-kernels
```

Comment:
```suggestion
#DISABLE_CUSTOM_KERNELS
```

Just for consistency with the rest, not sure if the parsing code cares about it or not.